README.md

```diff
@@ -172,6 +172,18 @@ You need to make sure to install the Crystal dependencies.
 
 > Run specific tests with `script/test <path_to_spec>`
 
+### Lucky to have you!
+
+We love all of the community members that have put in hard work helping to make Lucky better.
+If you're one of those people, we want to give you a t-shirt!
+
+Shirts are given to community members that have put in a ton of effort and contributions in to Lucky. 
```

Comment:
I think this sentence can probably be removed since I think the first 2 cover it